src/manage_migrators.c

```diff
@@ -2714,6 +2714,73 @@ migrate_243_to_244 ()
   return 0;
 }
 
+/**
+ * @brief Migrate the database from version 244 to version 245.
+ *
+ * @return 0 success, -1 error.
+ */
+int
+migrate_244_to_245 ()
+{
+  sql_begin_immediate ();
+
+  /* Ensure that the database is currently version 244. */
+
+  if (manage_db_version () != 244)
+    {
+      sql_rollback ();
+      return -1;
+    }
+
+  /* Update the database. */
+
+  sql ("ALTER TABLE reports RENAME COLUMN date TO creation_time;");
+
+
+  sql ("CREATE OR REPLACE FUNCTION task_last_report (integer)"
```

Comment:
Creating the functions should not be necessary here, as `manage_create_sql_functions` will be called when starting `gvmd` after the migration.